src/Sylius/Bundle/ResourceBundle/Controller/RequestConfiguration.php

```diff
@@ -557,4 +557,28 @@ public function getGrid()
 
         return $this->parameters->get('grid');
     }
+
+    /**
+     * @return bool
+     */
+    public function hasStateMachine()
+    {
+        return $this->parameters->has('state_machine');
+    }
```

Comment:
The function `hasStateMachine` should return something like this instead:
`return $this->parameters->get('state_machine[transition]', false, true);`
Because some cases would have no transition to continue and it may throw an error. 

For example, if an object is in the end of states, when we submit(update) the form to `updateAction` then the key `state_machine[transition]` will be null but the key `state_machine` will still exists.